CITATION

```diff
@@ -33,76 +39,257 @@ https://zenodo.org/record/3628791
       "@type": "Person", 
       "name": "Charles Overbeck"
     }, 
+    {
+      "@type": "Person", 
+      "name": "NatalieEO"
+    }, 
     {
       "affiliation": "UCSC Genomics Institute", 
       "@type": "Person", 
       "name": "Walter Shands"
     }, 
     {
+      "affiliation": "Ontario Institute for Cancer Research", 
       "@type": "Person", 
-      "name": "NatalieEO"
+      "name": "Gregory F J Hogue"
+    }, 
+    {
+      "@type": "Person", 
+      "name": "Emily Soth"
+    }, 
+    {
+      "@type": "Person", 
+      "name": "Louise Cabansay"
     }, 
     {
       "@type": "Person", 
       "name": "oicr-vchung"
     }, 
+    {
+      "@type": "Person", 
+      "name": "Andy Chen"
+    }, 
+    {
+      "@type": "Person", 
+      "name": "David Steinberg"{
+  "description": "Highlighted new features include\n<ul>\n<li>GitHub app support for filters to allow for more control over which tags and branches are automatically reported to Dockstore, more details <a href=\"https://docs.dockstore.org/en/develop/getting-started/github-apps/github-apps.html?highlight=filters\">here</a></li>\n<li>can select a specific version of a workflow or tool when adding then to a collection (\"I vouch for specifically version X of workflow Y\")</li>\n<li>cwltool updates from 1.0.20190621234233 to 3.0.20200807132242</li>\n<li>GitHub apps patches for more reliable processing of automatic updates</li>\n<li>performance improvements for large repositories with lots of branches and tags</li>\n<li>performance improvments for GA4GH TRS implementation</li>\n<li>a start to a more consistent visual experience for the site</li>\n<li>Behind the scenes<ul>\n<li>a large number of deployment and architectural changes to facilitate better security and regulatory compliance</li>\n<li>updates to elastic search and angular</li>\n</ul>\n</li>\n<li>a huge host of bugfixes for everything from WDL parsing, UI display issues, editing of hosted workflows, and command-line interface fixes \nSee a full list of our changes on <a href=\"https://github.com/dockstore/dockstore/milestone/37\">GitHub</a></li>\n</ul>\nBreaking changes\nMajor\n<ul>\n<li>none known (or intended anyway)</li>\n</ul>\nMinor\n<ul>\n<li>none</li>\n</ul>", 
```

Comment:
This does not seem right... Shouldn't there be a closing brace after Steinberg"?

Reply:
Hmmm, seems like a copy-paste error, retrying